src/main/java/vention/pages/BasePage.java

```diff
@@ -0,0 +1,73 @@
+package vention.pages;
+
+import vention.configLoader.ConfigLoader;
+import vention.driver.DriverManager;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.By;
+import java.time.Duration;
+
+public abstract class BasePage {
+  private static final int DEFAULT_TIME = 10;
+
+  public BasePage() {
+    PageFactory.initElements(DriverManager.getDriver(), this);
+  }
+
+  public String getRelativePath() {
+    return "";
+  }
+
+  public void openPage() {
+    String fullUrl = getBaseUrl() + getRelativePath();
```

Comment:
I'd recommend not to create a variable if we use the statement once and it's not too large.